guides/whitelist.htm

```diff
@@ -193,11 +212,31 @@ <h3>Greylist</h3>
                                       ❌
                                     {% endif %}
                                 </td>
+                                <td {% if project.gdpr == "yes" %}class="table-color-gdpr-yes"{% endif %}>
+                                    {% if project.gdpr == "yes" %}
+                                        <a href={{project.gdpr-enable-steps}}><span class="oi oi-warning"></span></a>
+                                    {% endif %}
+                                </td>
                             </tr>
                         {% endfor %}
                     </tbody>
                 </table>
             </div>
         </div>
+        <h3 id="about-gdpr" class="anchor-adjust">About GDPR</h3>
+            <p>GDPR is a law in the European Union for data privacy. Some projects have
+            interpreted it to mean that to you have to opt in for sharing any BOINC 
+            stats to other people. To earn Gridcoin you must enable this because
```

Comment:
>to other people
Suggestion: to third parties

Reply:
I thought about doing it that way, but I think "other people" would be a bit easier to understand especially since we have a lot of non-native English readers